src/dnx.mono.managed/EntryPoint.cs

```diff
@@ -58,6 +58,19 @@ public static int Main(string[] arguments)
         // Set the default lib path to be next to the entry point location
         Environment.SetEnvironmentVariable(EnvironmentNames.DefaultLib, Path.GetDirectoryName(typeof(EntryPoint).Assembly.Location));
         Environment.SetEnvironmentVariable(EnvironmentNames.ConsoleHost, "1");
+        
+        var p = (int)Environment.OSVersion.Platform;
+        string os;
+        if ((p != 4) && (p != 6) && (p != 128))
```

Comment:
I found: `It means Unix under the 1.x .NET profile;` - so we don't have to check against 128